vxsandbox/resources/metrics.py

```diff
@@ -75,10 +78,35 @@ def from_command(cls, command):
 
 
 class MetricsResource(SandboxResource):
-    """Resource that provides metric storing."""
+    """Resource that provides metric storing.
+
+    TODO: Describe how to set metrics_prefix.
+    #         return "%scampaigns.%s.stores.%s." % (
+    :param dict metrics_prefix:
+        metrics prefix configuration.
```

Comment:
I think that this docstring is incorrect. Looking at the code, it seems like this should be a string that describes the metrics prefix, not a dict.